### DISCUSSION TITLE
apollo单节点部署如何升级为集群模式

### DISCUSSION THREAD

**Discussion by git1e** [apolloconfig/apollo#3997] [Q&A]
apollo单节点部署已经运行一段时间，如何升级为集群模式。可不可以先部署个apollo集群，后端数据库也使用单节点的apollo数据库。然后切换代码中的configService（域名:port）地址到apollo集群上

**Comment by nobodyiam** [comment]
apollo-client 始终是从配置的 meta server 地址获取 config service 地址，只要确保升级过程中 meta server 地址可以访问，并且能否返回正确的 config service 地址即可。

**Comment by git1e** [comment]
新集群的apollo可以和老的单节点apollo共用同一个ApolloConfigDB和ApolloPortalDB么。如果可以的话，apollo-client中修改meta server 指向新的apollo集群的meta server。当所有的应用都切换到新apollo集群后，停掉老的apollo单节点。这样升级可行么

**Reply by nobodyiam** [reply]
数据库可以共用的，而且也应该共用~
